### DISCUSSION TITLE
How long does it take an author to add their article?

### DISCUSSION THREAD

**Discussion by jvwong** [PathwayCommons/factoid#1219] [Show and tell]
Purpose
===

Try to answer, with evidence: How long does it take for the typical author to add their article? 

Methods
===

- Documents
  - Sample: 293 of the 307 author-submitted ('public') documents (October 10, 2023)
  - NB: There may have been a few initial submissions where we didn't even have these status back in 2021
- Elapsed time
  - The time between Document status 'initiated' to 'submitted'
  - NB: Information was drawn from '_ops' database field which tracks operations on documents 
- Interactions
  - NB: Included intra-complex    

Results
===

**Table: Summary statistics**
|Stat |elapsedMin|numInteractions|
|------|----------|---------------|
|Min   |0.2       |0.0            |
|Max   |543498.2  |43.0           |
|Median|6.9       |2.0            |
|Mean  |4841.5    |3.1            |

**Figure 1. Distribution of interactions per document**
![Histogram of numInteractions](https://github.com/PathwayCommons/factoid/assets/4706307/e2133962-1b26-44eb-8111-dd1685db2c32)

**Figure 2. Distribution of elapsed time per document**
![Histogram of elapsedMin](https://github.com/PathwayCommons/factoid/assets/4706307/269496ca-95d8-4f30-9caf-0d423d4d4196)

**Comment by jvwong** [comment]
Rethinkdb query: [db_create2submit.txt](https://github.com/PathwayCommons/factoid/files/13228482/db_create2submit.txt)

**Comment by jvwong** [comment]
Some summary stats:

|FIELD1         |Min|Lower Quartile|Median|Upper Quartile|Max     |IQR |Lower Bound|Upper Bound|COUNT|
|---------------|---|--------------|------|--------------|--------|----|-----------|-----------|-----|
|elapsedMin     |0.2|3.3           |6.9   |15.7          |543498.2|12.4|0.0        |34.4       |242  |
|numInteractions|0.0|1.0           |2.0   |4.0           |43.0    |3.0 |0.0        |8.5        |242  |

And after removing outliers:

![chart](https://github.com/PathwayCommons/factoid/assets/4706307/857fe7d6-72dc-4f0e-aecf-2414f226c82b)
![chart (1)](https://github.com/PathwayCommons/factoid/assets/4706307/4d682ce1-9f0b-40fe-9609-9146ca60a52f)
![Histogram of numInteractions](https://github.com/PathwayCommons/factoid/assets/4706307/e35d6279-4ada-47f3-9649-87c28146dd02)

**Reply by maxkfranz** [reply]
Here’s how I interpret the numbers at first glance:

(1) The median is around 3 minutes.  Most of the area of the curve looks to be left of 6 minutes.

(2) There’s a long tail.  Some people create giant figures with lots of interactions, though most don’t.  Some people start drawing a figure, get distracted (e.g. by email or by getting a coffee) and come back to finish the figure.  I examined some of the longer ones in the admin panel, and the ones I saw matched this take-a-break-and-come-back pattern with the timing of the editor actions/operations.
